app/packs/src/apps/mydb/elements/details/samples/SampleDetails.js

```diff
@@ -951,13 +951,15 @@ export default class SampleDetails extends React.Component {
       />
     ) : null;
 
+    const inventoryLabel = sample.inventory_sample && sample.xref.inventory_label ? sample.xref.inventory_label : null;
```

Comment:
would fit better as  method of the  Sample classe (models/Sample.js)

Reply:
done